datagrepper/app.py

```diff
@@ -280,9 +280,8 @@ def raw():
     start = flask.request.args.get("start", None)
     end = flask.request.args.get("end", None)
     delta = flask.request.args.get("delta")
-    start, end, delta = assemble_timerange(
-        start, end, delta, app.config["DEFAULT_QUERY_DELTA"]
-    )
+    with app.app_context():
```

Comment:
The app context is automatically created when flask starts up, it should be done here. If you're getting errors in the unit tests because of the uninitialized flask app context, either use the testing client or create the app context in the tests. [Here are the docs about it](https://flask.palletsprojects.com/en/2.0.x/testing/#other-testing-tricks).